config/daemon/systemd.md

```diff
@@ -96,18 +96,34 @@ you need to add this configuration in the Docker systemd service file.
     ```
 
 3.  If you have internal Docker registries that you need to contact without
-    proxying you can specify them via the `NO_PROXY` environment variable:
-
+    proxying you can specify them via the `NO_PROXY` environment variable.
+
+    The `NO_PROXY` variable specifies a string that contains comma-separated values for hosts that should be excluded from proxying. 
```

Comment:
I think we don't need this bullet to be one level deeper; and we may be able to get rid of the code _block_ below.

I gave this a quick go, and think something like this might work:

```suggestion
    The `NO_PROXY` variable specifies a string that contains comma-separated
    values for hosts that should be excluded from proxying. These are the
    options you can specify to exclude hosts: 
    * IP address prefix (`1.2.3.4`) or in CIDR notation (`1.2.3.4/8`)    
    * Domain name, or a special DNS label (`*`)
    * A domain name matches that name and all subdomains. A domain name with
      a leading "." matches subdomains only. For example, given the domains
      `foo.example.com` and `example.com`;
      * `foo.com` matches `foo.com` and `bar.foo.com`, and
      * `.foo.com` matches only `bar.foo.com`
    * A single asterisk (`*`) indicates that no proxying should be done
    * Literal port numbers are accepted by IP address prefixes (`1.2.3.4:80`)
      and domain names (`foo.example.com:80`)
```

Which will look something like this;

The `NO_PROXY` variable specifies a string that contains comma-separated
values for hosts that should be excluded from proxying. These are the
options you can specify to exclude hosts: 
* IP address prefix (`1.2.3.4`) or in CIDR notation (`1.2.3.4/8`)    
* Domain name, or a special DNS label (`*`)
* A domain name matches that name and all subdomains. A domain name with
  a leading "." matches subdomains only. For example, given the domains
  `foo.example.com` and `example.com`;
  * `foo.com` matches `foo.com` and `bar.foo.com`, and
  * `.foo.com` matches only `bar.foo.com`
* A single asterisk (`*`) indicates that no proxying should be done
* Literal port numbers are accepted by IP address prefixes (`1.2.3.4:80`)
  and domain names (`foo.example.com:80`)

Reply:
Oh! I see there's a mis-match between the domains; updating my comment

Reply:
Done 👍